src/styles.scss

```diff
@@ -27,3 +27,8 @@ body {
   margin: 0;
   font-family: Roboto, "Helvetica Neue", sans-serif;
 }
+
+.mat-icon-button.mat-button-base {
```

Comment:
I think it would be better to move such styles to separate file. It's possible to create a folder with `overrides` name and `button.scss` or `mat-button.scss`. And than just import the created file.

Reply:
Good idea!